services/catarse/catarse.js/legacy/src/c/project-insights.js

```diff
@@ -76,7 +76,7 @@ const projectInsights = {
                 m(`input[type="hidden"][value="${contribution.total_contributed}"`),
                 'R$ ',
                 h.formatNumber(contribution.total_contributed, 2, 3),
-                m('span.w-hidden-small.w-hidden-tiny', ` (${contribution.total_on_percentage.toFixed(2)}%)`)
```

Comment:
@gabrielras eu não entendi pq remover essa tag.

Reply:
quando vc passa uma tag, o componente que ordena a tabela acaba ordenando ele tbm, então acaba que os valores ficam  errados, quando remove a tag e deixa apenas um string, ele filtra normalmente.

Reply:
@gabrielras mas removendo isso não altera o estilo do texto? Caso altere, não teria como só considerar o conteúdo da tag no momento de ordernar?
E vi que o conteúdo tem um espaço e um parentêse (` (`), isso não feta a ordenação também? Pois considerado que o espaço e o parentêse não afeta a ordenação, um `<span class='w-hiiden-small w-hidden-tiny> (` não deveria alterar também, já que todos terão essa mesma classe no início.

Reply:
@stephannv eu tinha feito o teste de com e sem o span, e não teve alteração no visual, mas oq acontece com a adição do span e um valor é que ele se comporta como se fosse outra coluna e tem uma ordenação própria, mas quando eu dou console.log(row), ele ordena normalmente e está com os valores corretos de porcentagem acompanhando os valores em real.